src/redux/store.js

```diff
@@ -1,12 +1,22 @@
 import { createStore, combineReducers } from 'redux';
 import todos from './modules/todos';
 import filterTodos from './modules/filter-todos';
+import { loadState, saveState } from '../localStorage';
+import throttle from 'lodash/throttle';
```

Comment:
It looks like an overkill to add the whole `lodash` library just to use the `throttle`. Are you sure that there is no other option? Also, can you check if only this module is bundled or the whole lodash?